src/Colors.ts

```diff
@@ -11,16 +12,19 @@ const {
   interpolate,
   divide,
   sub,
-  color,
+  color: reColor,
   Extrapolate
 } = Animated;
 
 interface RGBColor {
```

Comment:
So we can remove this type now no? Thanks for doing this PR!

Reply:
I don't think so, it's still being used on RGB -> HSV conversion... I don't know what's the use case for HSV color space, so I didn't want to remove it...

Thanks

Reply:
oh right, just saw it's private. Will review this PR in more depth soon.